cpp/src/tests/gdf_graph/gdf_graph.cu

```diff
@@ -243,21 +243,21 @@ TEST(gdf_delete_adjacency_list, success1)
       
   gdf_graph G;
   gdf_column col_off, col_ind, col_w;
-  size_t free, free2, total;  
-  cudaMemGetInfo(&free, &total);
+  //size_t free, free2, total;  
+  //cudaMemGetInfo(&free, &total);
```

Comment:
why comment out lines whether than removing them?

Reply:
In my description above I mentioned that it was unclear to me whether we want to not check this information at all, or if there was some RMM calls we could make that would be equivalent.  I left the commented lines in to know where we had those error checks.

I'm find deleting them instead, I just didn't want to delete something from the unit tester that someone really thinks we should be checking.